Application/Telegram/TelegramCommunicationService.cs

```diff
@@ -29,15 +30,9 @@ public TelegramCommunicationService(IConfiguration configuration, IMediator medi
 
         public async Task Execute(Update update)
         {
-            //for testing
-            if (update.Message !=null)
-            {
-                if (UserStateStorage.GetUserCurrentState(update.Message.From.Id) == UserState.ProcessNotStarted)
-                {
-                    UserStateStorage.AddUser(5213829376, UserState.ProcessNotStarted, UserRole.User);
-                }
-            }
-            
+            if (update.Message != null && !await Authentication.Authenticate(update, _mediator))
+                return;
```

Comment:
Could we add some message to user that he is not added to users and he will no able to use bot?

Reply:
Done. I added not-authorized message.